WireNetwork/Sources/WireNetwork/Network/NetworkService/NetworkService+Make.swift

```diff
@@ -0,0 +1,102 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension NetworkService {
+
+    static func makeServices(
+        backendConfig: BackendEnvironment2.Config,
+        minTLSVersion: TLSVersion,
+        proxyCredentials: ProxyCredentials? = nil
+    ) throws(InitializationError) -> (
+        rest: NetworkService,
+        webSocket: NetworkService
+    ) {
```

Comment:
thought: I wonder if that would make sense to have another method to return only the websocket NetworkService. i.e in the NSE you would only need it... it depends when you call it.